### PR TITLE
session is refreshed on payment post request

### PR DESCRIPTION
**Pull request by maxwofford**
@zachlatta This should fix the refresh errors on payment.

### CODE REVIEW

**Comment by maxwofford** [comment]
@zachlatta Actually, not finished testing --hold off on merging

**Comment by maxwofford** [comment]
@zachlatta I've done a few (bad) tests.

EDIT: everything passed though

File: unrejector/server.rb [hunk 1]
```diff
@@ -84,6 +80,15 @@ def payment_submitted(secret, spreadsheet)
     @amount = adjusted_cost ? adjusted_cost.to_i : 59500
     @paid = redis.get(@secret) == 'paid'
 
+    # Creates a session.
+    auth.fetch_access_token!
+    session = GoogleDrive.login_with_oauth(auth.access_token)
+    spreadsheet = session.spreadsheet_by_key(settings.spreadsheet_key)
+
+    Stripe.api_key = settings.stripe_secret_key
```
Comment (zachlatta) [inline unrejector/server.rb]: Do we need to set `Stripe.api_key` every time someone pays?

File: unrejector/server.rb [hunk 2]
```diff
@@ -84,6 +80,15 @@ def payment_submitted(secret, spreadsheet)
...
+
+    redis = Redis.new(url: settings.redis_url)
```
Comment (zachlatta) [inline unrejector/server.rb]: Do we need to reinitialize the Redis connection every time someone pays?

**Comment by zachlatta** [comment]
Left a few comments.

**Comment by maxwofford** [comment]
@zachlatta Thanks for that, yanked a few extra lines before committing on accident.

**Pull request merged**